conf/application.conf

```diff
@@ -119,7 +111,7 @@ metrics {
   rateUnit = SECONDS
   durationUnit = SECONDS
   showSamples = true
-  jvm = true
+  jvm = false
```

Comment:
ensure this is set on in Base, inform a QA that metrics reporting may have changed

Reply:
Done